rust/hyperlane-base/src/metrics/core.rs

```diff
@@ -1,9 +1,8 @@
 use std::collections::HashMap;
 use std::fmt::{Debug, Formatter};
-use std::sync::Arc;
+use std::sync::{Arc, OnceLock};
```

Comment:
nitpick: Do we actually need `OnceLock`? The stdlib provides its own `OnceCell` and I was wondering why not switch to that instead

Reply:
This is that. `std::cell::OnceCell` is declared `!Sync` which means we need `std::sync::OnceLock` which is equivlent to the old `once_cell::OnceCell`